spec/erb_lint/cli_spec.rb

```diff
@@ -4,7 +4,6 @@
 require "spec_utils"
 require "erb_lint/cli"
 require "erb_lint/cache"
-require "pp"
```

Comment:
This is needed. Somehow the load order is now different, so this now runs into https://github.com/fakefs/fakefs/issues/215